src/index.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const { git, gitMultilineResults, toTitleCase } = require('./utils');
+
+let packageName = JSON.parse(fs.readFileSync('./package.json').toString()).name;
+
+const packageNameAliases = {
+	'balena.io': 'resin-api',
+};
+
+packageName = packageNameAliases[packageName] || packageName;
+
+const moduleAliases = {
+	'resin-ui': 'dashboard',
+};
+
+const moduleName =
+	moduleAliases[packageName] ||
+	packageName.replace('resin-', '').replace('balena-', '');
+
+function getLatestProdInfo() {
+	const tags = gitMultilineResults('tag -l --sort=refname production-*');
+	let latestProdTag = tags[tags.length - 1];
+
+	// if the current commit has the production tag skip it
+	const currentCommitHash = git('rev-list -n 1 HEAD');
+	const latestProdTagCommitHash = git(`rev-list -n 1 ${latestProdTag}`);
+
+	if (currentCommitHash === latestProdTagCommitHash) {
```

Comment:
If the current commit has the production tag, doesn't that mean there is nothing to release?

Reply:
Oh, I see. One case where it might not be desirable is if two people fail to synchronize who will request a deploy, and they both end up requesting a deploy to the same version with a duplicated change-log. Considering we ping everyone involved, this is quite unlikely, so it's fine as it is, no need to overthink it at this point.

Reply:
Why do we set latestProdTag the the previous one in this case?

Reply:
@nazrhom  As I briefly mentioned earlier, this supports allows us to generate release notes even after a deploy reaches prod. It can be handly in scenarios like: someone requests a deploy w/o using this module and after the deploy is done they are reminded/instructed to post release notes by using this script.

We normally expect a tree like:
```
* HEAD, master
|\
| * PR
|/
* production, production-2019-01-15
```
but in the above case the tree would like like:
```
* HEAD, master, production, production-2019-01-16
|\
| * PR
|/
* production-2019-01-15
```
So, in order to be able to re-create the missed release notes, we just don't include the latest `production-*` tag in our algorithm.

It looks like I should add some comments about this :)